timesketch/frontend-ng/src/components/SettingsDialog.vue

```diff
@@ -0,0 +1,76 @@
+<!--
+Copyright 2024 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-card class="pa-4" min-height="800px">
+    <v-card-title>Settings</v-card-title>
+    <v-list v-if="settings" two-line subheader flat>
```

Comment:
Align title and content: 
![image](https://github.com/google/timesketch/assets/99879757/5b135595-e88b-497f-9970-4ddc478504cb)

```suggestion
    <v-list class="px-3" v-if="settings" two-line subheader flat>
```